src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -24,13 +24,7 @@
 import bio.terra.workspace.generated.model.ApiJobReport;
 import bio.terra.workspace.generated.model.ApiJobReport.StatusEnum;
 import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
-import bio.terra.workspace.service.job.exception.DuplicateJobIdException;
-import bio.terra.workspace.service.job.exception.InternalStairwayException;
-import bio.terra.workspace.service.job.exception.InvalidResultStateException;
-import bio.terra.workspace.service.job.exception.JobNotCompleteException;
-import bio.terra.workspace.service.job.exception.JobNotFoundException;
-import bio.terra.workspace.service.job.exception.JobResponseException;
-import bio.terra.workspace.service.job.exception.JobUnauthorizedException;
+import bio.terra.workspace.service.job.exception.*;
```

Comment:
Standard practice is no `*`. You have to make settings in IntelliJ for it to not do that. I can give you the info if you need it.